fwprovider/provider.go

```diff
@@ -193,9 +195,9 @@ func (p *Provider) Configure(ctx context.Context, req tfprovider.ConfigureReques
 // Resources returns the resources supported by the provider.
 func (p *Provider) Resources(ctx context.Context) []func() resource.Resource {
 	return []func() resource.Resource{
-		NewClusterWaiterResource,
-		NewDNSDomainResource,
-		NewClusterRosaClassicResource,
+		clusterwaiter.NewClusterWaiterResource,
+		dnsdomain.NewDNSDomainResource,
+		// clusterwaiter.NewClusterRosaClassicResource,
```

Comment:
why is it commented-out?

Reply:
Because its still not working
It need to be uncomment after @bardielle fix

Reply:
coolz
can you add a TODO at least?

Reply:
Done